plugin.video.arteplussept/resources/language/English/strings.po

```diff
@@ -0,0 +1,35 @@
+# Kodi Media Center language file
```

Comment:
Why are there duplicate language files? `English` and `resource.language.en_gb `. Same goes for all languages?

Reply:
I think this is leftovers from previous a translation system. I'll clean it up.

Reply:
Done